README.md

```diff
@@ -135,10 +135,10 @@ docker run --rm \
 ```
 
 - Launch spark-shell with ARLAS-PROC dependency
-```
+```bash
 docker run -ti \
        -w /opt/work \
-       -v ${PWD}/data/ais:/opt/data \
+       -v ${PWD}/data/ais/data:/opt/data \
```

Comment:
To be fair, this must be done in the same commit where you move the data directory. BTW, it's not a big issue in that context.